src/App.vue

```diff
@@ -1,19 +1,14 @@
 <template>
   <div class="h-screen overflow-hidden grid grid-cols-[70px,auto] divide-x">
-    <NavigationBar>
-      <template v-slot:nav-icon>
-        <IconLogin />
-      </template>
-    </NavigationBar>
+    <NavigationBar> </NavigationBar>
```

Comment:
just use "<NavigationBar />" instead of "<NavigationBar> </NavigationBar" if there is nothing to add inside